Core/Core/SwiftUIViews/EditorForm.swift

```diff
@@ -92,15 +118,42 @@ public struct TextFieldRow: View {
     }
 
     public var body: some View {
-        HStack {
+        EditorRow {
             label
-                .font(.semibold16).foregroundColor(.textDarkest)
                 .accessibility(hidden: true)
+            Spacer()
             TextField(placeholder, text: $text)
                 .multilineTextAlignment(.trailing)
                 .font(.regular16).foregroundColor(.textDarkest)
                 .accessibility(label: label)
         }
-            .padding(16)
     }
 }
+
+public struct DoubleFieldRow: View {
+    public let label: Text
+    public let placeholder: String
+    @Binding public var value: Double?
+
+    public init(label: Text, placeholder: String, value: Binding<Double?>) {
+        self.label = label
+        self.placeholder = placeholder
+        self._value = value
+    }
+
+    public var body: some View {
+        TextFieldRow(label: label, placeholder: placeholder, text: Binding(
```

Comment:
TextField can take a binding & formatter that theoretically does the same thing, but I found that it won't update the binding on each keystroke, so I'd often hit the done button & the updated value I had typed did not get saved. Making our own parsing binding fixes that.

Reply:
Yikes. That's a scary bug.